roles/docker_storage_to_overlay2/vars/main.yml

```diff
@@ -1,16 +1,15 @@
 ---
-storage_setup_path: "/etc/sysconfig/docker-storage-setup"
+docker_storage_to_overlay2_storage_setup_path: "/etc/sysconfig/docker-storage-setup"
 
 # List of services to stop and restart.  Any available services
 # in optional_service_names get appended to this list beforehand.
-service_names: []
+docker_storage_to_overlay2_service_names:
+- "{{ openshift.docker.service_name }}"
 
-optional_service_names:
-- "docker.service"            # } One or the other
-- "container-engine.service"  # } must be present
+docker_storage_to_overlay2_optional_service_names:
```

Comment:
Note that when ``origin`` is installed the service is called ``origin-node.service`` You can use ``{{ openshift.common.service_type }}`` to get the proper one.